src/_cffi_src/build_openssl.py

```diff
@@ -19,10 +19,20 @@ def _get_openssl_libraries(platform):
             os.environ.get("CRYPTOGRAPHY_OSX_NO_LINK_FLAGS")
         )
     elif platform == "win32":
-        if compiler_type() == "msvc":
-            libs = ["libeay32", "ssleay32"]
+        # A space separated list of libraries to link against. With MSVC this
+        # needs to be the full name of the lib, e.g. libcrypto, not crypto
+        lib_names = os.environ.get("CRYPTOGRAPHY_WINDOWS_LIBRARIES", None)
```

Comment:
To switch between the 1.1.0 and above names and the older names? We could do it that way too. Then we'd probably want to do `if compiler_type() == "msvc" and not os.environ.get("CRYPTOGRAPHY_OPENSSL_110")`? Of course, that flag will still only affect windows.